nodeup/pkg/model/ntp.go

```diff
@@ -46,22 +57,103 @@ func (b *NTPBuilder) Build(c *fi.ModelBuilderContext) error {
 		return nil
 	}
 
+	var ntpIP string
+	switch b.Cluster.Spec.CloudProvider {
+	case "aws":
+		ntpIP = "169.254.169.123"
+	case "gce":
+		ntpIP = "time.google.com"
```

Comment:
I think you are right. I got the information from this doc https://developers.google.com/time/faq but it looks like time.google.com is public for all people to use outside of GCP.

I can't resolve metadata.google.internal from my laptop but I assume it will when running on GCP. 

Shall I update?